setup.cfg

```diff
@@ -22,7 +22,8 @@ data_files =
     /usr/share/ansible/roles/ipareplica = roles/ipareplica/*
 
 [flake8]
-extend-ignore = E203
+extend-ignore = E203, D1, D212, D203, D400, D401
```

Comment:
Why did you need to ignore D400 and D401 as well?

Reply:
In my local setup, flake8 complains about D400 and D401 (I have flake8-docstrings 1.5.0 and pydocstyle 5.0.1), and pydocstyle didn't. I wanted to have the same output on both, and lead by pydocstyle.